modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -301,6 +301,11 @@ If you opted to use private API endpoints, you must use an existing VPC and Priv
 ====
 You must ensure that your VPC is configured with a public and a private subnet for each availability zone that you want the cluster installed into. If you opted to use PrivateLink, only private subnets are required.
 ====
+.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the default machine pools. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
```

Comment:
Now the block "Additional security groups" does not only apply to the machine pools, but also to the infra and control plane nodes.

So the sentence "select additional custom security groups to apply to nodes in **the default machine pools**" doesn't seem accurate.

Reply:
@celdrake @gdbranco This is in the "create cluster" flow, so this part controls only the machine pools created by default, right? Should I refer to those in some way other than "default machine pools"?

Reply:
So, personally I like machine pools as the term simply works. But the infra and control plane nodes are not referenced with this term. So it might end up being slightly confusing.

Reply:
Updated to "machine pools created by default".